src/core/components/layout/components/columns/columns.tsx

```diff
@@ -51,13 +54,16 @@ const Columns = ({
 	collapseBelow,
 	cssOverrides,
 	children,
+	spaceY = 5,
```

Comment:
This preserves the current behaviour by default but means that the default behaviour is different to the `Stack` component. I'm not sure which is preferable.

Reply:
I think we ought to replicate Stack behaviour here for consistency. We can afford to make a "breaking" change because nobody is using this behaviour yet. Arguably zero space between rows is the correct and expected behaviour.

Reply:
Cool, agreed. 👍